vital/types/metadata_traits.h

```diff
@@ -49,6 +49,11 @@ TAG_TYPE( VITAL_META_VIDEO_URI, string_t );
 TAG_TYPE( VITAL_META_VIDEO_KEY_FRAME, bool );
 TAG_TYPE( VITAL_META_VIDEO_FRAME_NUMBER, uint64_t );
 TAG_TYPE( VITAL_META_VIDEO_MICROSECONDS, uint64_t );
+TAG_TYPE( VITAL_META_VIDEO_FRAME_RATE, double );
+TAG_TYPE( VITAL_META_VIDEO_BITRATE, uint64_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_TYPE, string_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_PROFILE, string_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_LEVEL, string_t );
```

Comment:
Do these need to be alphabetical?

Reply:
It doesn't seem like the rest of the list is trying for alphabetical, so I just went with putting related information next to each other (like how `IMAGE_WIDTH` and `IMAGE_HEIGHT` are next to each other).